charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -173,17 +173,15 @@ queries: {}
 #  ingestion_rate: 'sum(rate(vm_rows_inserted_total[5m])) by (type,accountID) > 0'
 #  insertion_rate: 'sum(rate(vm_http_requests_total{path=~"/api/v1/write|.*insert.*"}[5m])) by (path) > 0'
 #  slow_inserts: 'sum(rate(vm_slow_row_inserts_total[5m])) / sum(rate(vm_rows_inserted_total[5m]))'
-
+samplingPeriod: "1m"
 # -- vmanomaly internal monitoring in Prometheus format
 monitoring:
   # /metrics server.
   pull:
     # -- if true expose metrics on /metric endpoint
-    enabled: true
     port: 8440
   push:
     # -- push metrics on prometheus format if defined
-    endpoint_url: ""
+    url: ""
```

Comment:
NOTE, it looks like breaking change. You have to mention it and provide migration path.